src/main/resources/liquibase/scripts/scripts21.sql

```diff
@@ -0,0 +1,30 @@
+-- liquibase formatted sql
+
+-- changeset annaa:1
+
+CREATE TABLE notification_task (
+    id SERIAL unique PRIMARY KEY,
+    chat_id SERIAL,
+    text_msg TEXT,
+    notification_time TIMESTAMP
+);
```

Comment:
предлагаю убрать все changeset'ы и оставить только такой
```suggestion
-- changeset annaa:1

CREATE TABLE notification_task (
    id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
    chat_id BIGINT,
    text_msg TEXT,
    notification_time TIMESTAMP
);
```